unity/Assets/Scripts/GameState.cs

```diff
@@ -62,17 +62,25 @@ void Start()
 
     public void LevelWon()
     {
+        //Score.scoreValue += 10;
         Debug.Log("Level Complete!");
         lcdController.StopTimer();
         uiManager.SwitchWin();
         levelNumberText.text = "Level " + (levelNumber).ToString() + " Completed!";
         levelNumber += 1;
-        score = (int) lcdController.InitialTime;
-        scoreText.text = "Score" + score.ToString();
+        Debug.Log("This is the lcdcontroller initial time: "+ lcdController.InitialTime);
+        Debug.Log("This is the time.deltatime variable"+ Time.deltaTime);
+        Debug.Log("this is the score variable before:" + score);
+        
+        
+        score = (int)Mathf.Round((lcdController.InitialTime-timer)*100);
+        Debug.Log("This is the score variable after:" + score);
+        scoreText.text = "Score: " + score.ToString();
     }
 
     public void GameOver()
     {
+        Score.scoreValue += 10;
```

Comment:
Shouldn't be adding 10 to the score on gameover. The reason I had it in LevelWon was just to text it because the game was broken af.